packages/build/src/core/constants.js

```diff
@@ -73,6 +74,10 @@ const getConstants = async function ({
      * The Netlify Site ID
      */
     SITE_ID: siteId,
+    /**
+     * The Netlify API endpoint
+     */
+    NETLIFY_API_HOST: apiHost,
```

Comment:
This will be exposed to plugin authors, is this intentional?

Reply:
This is intentional because this would allow properly developed plugins to be tested in staging. It's not vital by any means, tho.

Reply:
Thanks for the explanation @Benaiah.
Should this be documented or hidden?

Reply:
I think this should be documented, since we'd ideally like plugin authors to respect it.

Reply:
I agree :+1: 
Should we create an issue for it in `netlify/docs`?

cc @rstavchansky @verythorough

Reply:
I'm interested to hear from @verythorough on whether we'd expect external Build Plugin developers to test against a staging server. If so, @Benaiah I'd appreciate it very much if you could create a docs issue to track this update.

Reply:
My thought was that we'd want to be able to test build plugins on staging for internal use, not that external authors would be using our staging system.

Reply:
We've decided to not expose this env variable for now and continue the discussion in https://github.com/netlify/build/issues/2373.